src/pages/library/Datasets.js

```diff
@@ -240,6 +243,39 @@ const amppd = () =>
     ]
   );
 
+const axin = () =>
+  h(
+    Participant,
+    {
+      logo: { src: axinLogo, alt: 'AnalytiXIN logo', height: '55%' },
+      title: 'AnalytiX Indiana Health Dataset',
+      description: h(Fragment, [
+        'The ',
+        h(Link, { href: 'https://analytixindiana.com', ...Utils.newTabLinkProps }, 'AnalytiXIN life sciences health data asset (HDA)'),
+        ` has been designed through a working
+        collaboration involving Eli Lilly and Company, IU Health, the Indiana Biobank within the
+        IU School of Medicine, the Indiana Health Information Exchange, and other partners to build
+        a shared health-data platform linking consented clinical and genomic patient data. This
+        collaboration is further leveraging the Broad Institute’s technical and genomic
+        infrastructure to accelerate the build-out as well as connect to broader national
+        initiatives such as the National Institutes of Health All of Us research program.`,
```

Comment:
Also, now that there is this description test, maybe we can have a ticket to update the overview page as well!

Reply:
I think both of these should get their description from the settings data. Not sure how well it would work to add an Ajax call to library page though.

Reply:
I made https://broadworkbench.atlassian.net/browse/DC-1189 to track this work.